component/common/application/matter/common/port/matter_wifis.c

```diff
@@ -338,9 +338,7 @@ int matter_wifi_connect(
 
     matter_wifi_trigger = 1;
     matter_set_autoreconnect(1);
-    wifi_connect(ssid, security_type, password, strlen(ssid), strlen(password), key_id, NULL);
-
-    return RTW_SUCCESS;
```

Comment:
@step0035 I had fixed the return value to success for ConnectNetwork() to pass and trigger autoreconnect. Otherwise, upon first connection failure (e.g. weak signal, wpa3, etc), error is detected and commissioning will be terminated.

Reply:
ok.
@SirRu24 , can you undo this changes?